src/main/java/commands/Command.java

```diff
@@ -0,0 +1,130 @@
+package commands;
+import exceptions.InvalidAddTaskException;
+import exceptions.InvalidCommandException;
+import exceptions.TaskListOutofBoundsException;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.Todo;
+import userInterface.Print;
+import duke.Duke;
+
+public class Command {
+
+    public static void throwInvalidCommand() throws InvalidCommandException{
+        throw new InvalidCommandException();
+    }
+
+    public static boolean isEventValid(String[] userInputArray){
+        if (userInputArray.length == 1){
+            return false;           //return false if task is empty
+        }
+        boolean fromExists = false;
+        boolean toExists = false;
+        boolean descriptionAndDatesExists = false;
```

Comment:
something like `containsFrom` or `hasFrom` etc would work betetr as it makes it more readable